### PR TITLE
netifyd: update to version 4.4.3

### PR DESCRIPTION
**Pull request by filippocarletti**
Newer version of the plugins are needed, to be found in branch vendor-nethesis-v4.4.x

### CODE REVIEW

**Comment by filippocarletti** [comment]
Note: a new builder is needed because of netifyd plugins.

File: packages/netify-flow-actions/Makefile [hunk 1]
```diff
@@ -15,9 +15,9 @@ PKG_INSTALL:=1
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL:=file:///home/build/openwrt/netify-flow-actions
-PKG_SOURCE_DATE:=2022-11-03
-#PKG_SOURCE_VERSION:=v1.0.1
-PKG_SOURCE_VERSION:=133fcc5340c95abc5cd5190726c236b68d443d62
+PKG_SOURCE_DATE:=2023-03-03
+#PKG_SOURCE_VERSION:=v1.0.12
```
Comment (gsanchietti) [inline packages/netify-flow-actions/Makefile:19]: We can cleanup this line:
```suggestion
```

File: packages/netify-flow-actions/Makefile [hunk 2]
```diff
@@ -42,6 +42,7 @@ define Package/netify-flow-actions
   DEPENDS+=+NETIFY_FLOW_ACTIONS_TARGET_CTLABEL:libnetfilter-conntrack
   DEPENDS+=+NETIFY_FLOW_ACTIONS_TARGET_IPSET:libipset
   DEPENDS+=+NETIFY_FLOW_ACTIONS_TARGET_NFTSET:nftables
+  EXTRA_DEPENDS:=netifyd (>=4.4.1-1)
```
Comment (gsanchietti) [inline packages/netify-flow-actions/Makefile:45]: Why is this needed?

File: packages/netify-plugin-stats/Makefile [hunk 3]
```diff
@@ -15,9 +15,9 @@ PKG_INSTALL:=1
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL:=file:///home/build/openwrt/netify-agent-stats-plugin
-PKG_SOURCE_DATE:=2022-09-07
-#PKG_SOURCE_VERSION:=v1.0.11
-PKG_SOURCE_VERSION:=6d3a37d9035ddf01b8c969b3b687fb8b0e3f6ab1
+PKG_SOURCE_DATE:=2023-03-03
+#PKG_SOURCE_VERSION:=v1.0.17
```
Comment (gsanchietti) [inline packages/netify-plugin-stats/Makefile:19]: We can cleanup also this one:
```suggestion
```

File: packages/netify-plugin-stats/Makefile [hunk 4]
```diff
@@ -32,6 +32,7 @@ define Package/netify-plugin-stats
   # std::unordered_map is only available via libstdcpp which is required for
   # performance reasons.
   DEPENDS+=+libstdcpp
+  EXTRA_DEPENDS:=netifyd (>=4.4.1)
```
Comment (gsanchietti) [inline packages/netify-plugin-stats/Makefile:35]: Even this one should not be needed: the plugin is part of the image.

File: packages/netifyd/Makefile [hunk 5]
```diff
@@ -16,9 +16,10 @@ PKG_INSTALL:=1
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL:=https://gitlab.com/netify.ai/public/netify-agent.git
-PKG_SOURCE_DATE:=2022-11-07
-PKG_SOURCE_VERSION:=189a993a831a3eb74951fc19ea54e6144263c09e
-PKG_MIRROR_HASH:=9762c98053fdc7f6f40560b6af9638cbe380dfc84201ffc91ec1e1005323b06c
+PKG_SOURCE_DATE:=2023-06-15
+PKG_SOURCE_VERSION:=v4.4.3
+#PKG_SOURCE_VERSION:=411df94c131e034d0f30ff6773e6871b23640f01
+PKG_MIRROR_HASH:=whatever
```
Comment (gsanchietti) [inline packages/netifyd/Makefile:22]: I'm not sure switching to the tagged release since upstream is not really accurate about versioning.

Still, why version 4.4.3 while upstream is 4.4.1 https://gitlab.com/netify.ai/private/nethesis/netify-flow-actions/-/commit/81ea8c4127238f230f919b2718ead07862d0a233?
```suggestion
PKG_SOURCE_DATE:=2023-06-15
PKG_SOURCE_VERSION:=v4.4.1
PKG_MIRROR_HASH:=whatever
```

**Review by gsanchietti — CHANGES_REQUESTED** [review]

**Review by gsanchietti — APPROVED** [review]
